app/javascript/packs/public.js

```diff
@@ -3,6 +3,7 @@ import { getLocale } from 'mastodon/locales';
 import { length } from 'stringz';
 import IntlRelativeFormat from 'intl-relativeformat';
 import { delegate } from 'rails-ujs';
+import loadPolyfills from '../mastodon/load_polyfills';
```

Comment:
Agreed, I'd rather it were just consistent everywhere. I'd be happy to see the alias removed assuming it's not too controversial. 🙂